app/models/project.rb

```diff
@@ -60,4 +69,15 @@ def related_projects
   def tasks_url
     "#{github_url}/issues"
   end
+
+  def self.parse_git_url(url)
+    url.gsub(/^(((https|http|git)?:\/\/(www\.)?)|git@)github.com(:|\/)/i, '')
+       .gsub(/(\.git|\/)$/i, '')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 3) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.